cmd/buildah/bud.go

```diff
@@ -203,6 +203,9 @@ func budCmd(c *cli.Context) error {
 		Runtime:             runtime,
 		RuntimeArgs:         runtimeFlags,
 	}
+	if !quiet {
+		options.ReportWriter = os.Stderr
```

Comment:
skopeo uses stdout, but we've previously been attempting to make it useful to capture the stdout of the `from` and `commit` commands, so stderr seemed to be a better choice.